share/picongpu/examples/LaserWakefield/include/picongpu/simulation_defines/param/density.param

```diff
@@ -24,6 +24,10 @@
 /* preprocessor struct generator */
 #include <pmacc/preprocessor/struct.hpp>
 
+// given in 1/m^3
+#ifndef UI_MACRO_DENSITY
```

Comment:
@codingS3b @jkelling contrary to what we discussed, let us please not use a `UI_MACRO_` prefix but a `PARAM_` prefix for these defines.
Please call the defines *exactly* like their variable counterparts, just with a `PARAM_` prefix (and also all caps).

The reason is, that also other defines use the same prefix already and it can be controlled / forwarded via a CMake switch in `-DPARAM_OVERWRITES:LIST`. So this would just be consistent.

Sorry for first passing the wrong message on that.